extensions/ql-vscode/src/view/results/result-tables.tsx

```diff
@@ -27,8 +27,7 @@ import {
 } from "./result-table-utils";
 import { vscode } from "../vscode-api";
 import { sendTelemetry } from "../common/telemetry";
-
-const FILE_PATH_REGEX = /^(?:.+[\\/])*(.+)$/;
+import { basename } from "path";
```

Comment:
Good catch! I'll replace with the `basename` function from `src/common/path`.

I didn't realise we were using `path-browserify`. It gave me a false sense of security because I was expecting it to fail to compile when referencing a nodejs module that is not unavailable in webviews.

Reply:
I think we might be able to get rid of `path-browserify`, there don't seem many instances of `path` being imported. I'll create an internal issue to track this.

Reply:
Thanks. I was also thinking that removing it would make things a bit safer and was wondering how easy it would be to remove it.